dask/tests/test_config.py

```diff
@@ -489,3 +491,22 @@ def test_get_override_with():
             "hello": "world"
         }
         assert dask.config.get("foo", override_with=["one"]) == ["one"]
+
+
+def test_config_serialization():
+    # Use context manager without changing the value to ensure test side effects are restored
+    with dask.config.set({"array.svg.size": dask.config.get("array.svg.size")}):
+
+        # Take a round trip through the serialization
+        serialized = serialize({"array": {"svg": {"size": 150}}})
+        config = deserialize(serialized)
+
+        dask.config.update(dask.config.global_config, config)
+        assert dask.config.get("array.svg.size") == 150
+
+
+def test_config_inheritance():
+    config = collect_env(
+        {"DASK_INTERNAL_INHERIT_CONFIG": serialize({"array": {"svg": {"size": 150}}})}
+    )
+    assert dask.config.get("array.svg.size", config=config) == 150
```

Comment:
Still want to unset this value after the test?

Reply:
Shouldn't need to, I am no longer affecting the global Dask config.

Reply:
oh, `config=` - sorry...